### ISSUE TITLE
SfRadialGauge annotation/gauge not centerable

### ISSUE THREAD

**Issue by obrunsmann** [syncfusion/flutter-widgets#800]
Hey Guys,

I am currently troubling with centering annotation/gauge widget:
![simulator_screenshot_2CFBA50B-8E55-4CD4-BBBB-A976D47DC946](https://user-images.githubusercontent.com/763812/174760219-7eb36c07-6ac7-4ac9-8308-d1dec6c4826f.png)

For some reason the gauge wrapper overflows to the left which while the Axis is still centered 🤯. But the annotation's anchor is the gauge container, so annotation is not centered.

```dart
Column(
  children: [
    Text(
      'Demo',
      textAlign: TextAlign.center,
      style: Theme.of(context).textTheme.headlineLarge,
    ).center(),
    const Spacer(),
    SfRadialGauge(
      animationDuration: 1500,
      enableLoadingAnimation: true,
      axes: [
        RadialAxis(
          showLabels: false,
          showTicks: false,
          minimum: 0,
          maximum: 10,
          startAngle: 270,
          endAngle: 270,
          axisLineStyle: const AxisLineStyle(
            color: widgetBackgroundColor,
            thickness: 20,
          ),
          pointers: [
            RangePointer(
              value: 10,
              width: 20,
              color: Theme.of(context).primaryColor,
              cornerStyle: CornerStyle.bothCurve,
              enableAnimation: true,
              animationType: AnimationType.ease,
              animationDuration: 400,
            )
          ],
          annotations: [
            GaugeAnnotation(
              widget: Text(
                '10',
                style: const TextStyle(
                  fontSize: 60,
                  height: 1,
                  fontWeight: FontWeight.w600,
                ),
              ).center(),
            )
          ],
        )
      ],
    ),
    const Spacer(),
  ],
)
```

Not sure what's going on here :/

**Comment by Yuvaraj-Gajaraj** [comment]
Hi @obrunsmann,

We have analyzed your code snippet and found that you did not mention the angle of the annotation. In default, it is null to achieve your requirement we suggest you set the annotation angle property value like 270 or 90 it will help you to place the annotation exact center of the screen. We have attached the code snippet below for your reference.

Code snippet:
```dart
annotations: const [
  GaugeAnnotation(
    angle: 270,
    widget: Text(
      '10',
      style: TextStyle(
        fontSize: 60,
        height: 1,
        fontWeight: FontWeight.w600,
      ),
    ),
  )
]
```

UG: https://help.syncfusion.com/flutter/radial-gauge/annotation#alignment-of-annotation 

Regards,
Yuvaraj.
